themes/default/content/docs/intro/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -23,19 +23,19 @@ Choosing the right infrastructure as code tool is important, and we want you to
 
 ## What is AWS CDK?
 
-AWS Cloud Development Kit (CDK) is an open-source software development framework for defining AWS cloud resources with general-purpose programming languages. CDK is a transpiler that converts program code into AWS Cloud Assembly, an intermediate format that is used to produce AWS CloudFormation template code and bundled resources for submission to AWS and the CloudFormation deployment service.
+AWS Cloud Development Kit (CDK) is an open-source software development framework for defining AWS cloud resources with general-purpose programming languages. CDK is a transpiler that converts program code into AWS CloudFormation JSON/YAML templates and other assets and then submits them to AWS and the CloudFormation deployment service.
 
 ## Pulumi vs. CDK: Similarities {#similarities}
 
-Both Pulumi and CDK allow you to build and deploy infrastructure on AWS using familiar programming languages and tools, and both use a declarative infrastructure as code model built on the concept of _desired state_. In both Pulumi and CDK, desired state is expressed by declaring one or more cloud resources in code that’s evaluated by the deployment engine, which in turn decides whether to create, update or destroy the resources of a given stack by comparing its last known state with the new state expressed in the code. Both have similar concepts for modularity and higher-level abstractions that encapsulate multiple resources. These are known as constructs in CDK and as Component Resources in Pulumi.
+Both Pulumi and CDK allow you to build and deploy infrastructure on AWS using familiar programming languages and tools, and both use a declarative infrastructure as code model built on the concept of _desired state_. In both Pulumi and CDK, desired state is expressed by declaring one or more cloud resources in code that’s evaluated by the deployment engine, which in turn decides whether to create, update or destroy the resources of a given stack by comparing the stack's last known state with the new state expressed in the code. Both have similar concepts for modularity and higher-level abstractions that encapsulate multiple resources. These are known as constructs in CDK and as Component Resources in Pulumi.
```

Comment:
aaron had commented before that we should caveat that CDK's desired state comes from CF

Reply:
Ah, ok. Well, hm -- this is the similarities section. Maybe a brief parenthetical would suffice -- I'll see if I can come up with something.

Reply:
@gthuang Take a look at my revisions -- I added a brief parenthetical to clarify that the code is rendered CFN text.